.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,32 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '17 2 * * 0'
```

Comment:
What is the significance of 2:17 on a Sunday?

Reply:
That's just a random weekly time picked

Reply:
Then perhaps 0:00 is as good a time as any? 2:17 just reads odd.

Reply:
Thanks. Updated the cron time.